app/models/filters/filter_base.rb

```diff
@@ -112,6 +116,7 @@ def update(filters) # rubocop:disable Metrics/CyclomaticComplexity, Metrics/Perc
       self.races = filters.dig(:races)&.select { |race| HUD.races(multi_racial: true).keys.include?(race) }.presence || races
       self.ethnicities = filters.dig(:ethnicities)&.reject(&:blank?)&.map(&:to_i).presence || ethnicities
       self.project_group_ids = filters.dig(:project_group_ids)&.reject(&:blank?)&.map(&:to_i).presence || project_group_ids
+      self.uses_default_project_ids = filters.dig(:project_group_ids).nil? || uses_default_project_ids
```

Comment:
can we move these up to just below `uses_default_project_ids` on line 110 above?

Reply:
I was grouping them with the user-visible attribute that potentially sets it... I'm more comfortable with it here but I can see the value of both options...

But yeah, it looks like I thought I might need it, and then decided it was better to check for an empty set of effective ids... So, I'll take this out and render the conversation moot :-)